### PR TITLE
Strict TLS 1.2 support for ALBs

### PR DESCRIPTION
**Pull request by adborden**
- Sort annotations to 🤞  reduce merge conflicts
- Use a TLS 1.2-only ALB policy.

https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html

### CODE REVIEW

**Comment by adborden** [comment]
oops, I guess the `/` in my branch needs to be slugified.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,25 @@ docs.
...
+Each brokered AWS EKS provides:
+
+- unique, isolated VPC with multiple AZs
+- a single unique namespace
+- Kubernetes notes provided by AWS EKS Fargate
+- RBAC
+- cluster-wide logging to AWS CloudWatch with fluent-bit
+- AWS [ALB Ingress Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/) for cluster-wide ingress, WAFv2 integration
+- nginx Ingress controller
+- AWS Route53 integration with [ExternalDNS](https://github.com/kubernetes-sigs/external-dns)
```
Comment (mogul) [inline README.md:31]:
```suggestion
- Automatic DNS configuration using AWS Route53 [via ExternalDNS](https://github.com/kubernetes-sigs/external-dns)
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,25 @@ docs.
...
+- [ZooKeeper Operator](https://github.com/pravega/zookeeper-operator) for
```
Comment (mogul) [inline README.md:32]:
```suggestion
- [ZooKeeper CRDs](https://github.com/pravega/zookeeper-operator) ready for
```

File: README.md [hunk 3]
```diff
@@ -16,6 +16,25 @@ docs.
...
+  managing Apache ZooKeeper clusters
+- [Solr Operator](https://github.com/apache/solr-operator) for managing
```
Comment (mogul) [inline README.md:34]:
```suggestion
- [Solr CRDs](https://github.com/apache/solr-operator) for managing
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -46,8 +46,8 @@ jobs:
       # Docs for this method:
       # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
       run: |
-        echo "INSTANCE_NAME=${GITHUB_HEAD_REF:0:20}-${GITHUB_RUN_ID}" >> $GITHUB_ENV
-        grep INSTANCE_NAME $GITHUB_ENV
+        INSTANCE_NAME="$(echo ${GITHUB_HEAD_REF:0:20}-${GITHUB_RUN_ID} | tr [:upper:] [:lower:] | sed -e 's/[^a-z0-9-]\+/-/g')"  # lowercase, slugify
```
Comment (mogul) [inline .github/workflows/test.yml:49]:
```suggestion
        INSTANCE_NAME="$(echo ci-${GITHUB_REF#refs/*/}-${GITHUB_RUN_ID})"
```

**Review by mogul — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -46,8 +46,8 @@ jobs:
       # Docs for this method:
       # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
       run: |
-        echo "INSTANCE_NAME=${GITHUB_HEAD_REF:0:20}-${GITHUB_RUN_ID}" >> $GITHUB_ENV
-        grep INSTANCE_NAME $GITHUB_ENV
+        INSTANCE_NAME="$(echo ci-${GITHUB_REF#refs/*/}-${GITHUB_RUN_ID})"
```
Comment (mogul) [inline .github/workflows/test.yml:49]:
```suggestion
        INSTANCE_NAME="$(echo ci-${{ github.event.pull_request.number }}-${GITHUB_RUN_ID})"
```

**Review by mogul — APPROVED** [review]

**Pull request merged**